automod2/automod2.py

```diff
@@ -304,20 +304,22 @@ async def mod_message_images(self, message):
             user_logs = self.channel_user_logs[key]
             count = 0
             for m in user_logs:
-                count += linked_img_count(m)
+                if abs((m.created_at - datetime.utcnow()).total_seconds()) < 300: # only check messages in past 300 seconds
```

Comment:
There's no need for the abs because `datetime.utcnow()` will always be more than `m.created_at`.  Just switch the minuend and the subtrahend.